src/ivc/fold_relaxed_plonk_instance_chip.rs

```diff
@@ -809,11 +811,78 @@ mod tests {
 
     type Base = <C1 as CurveAffine>::Base;
     type ScalarExt = <C1 as CurveAffine>::ScalarExt;
+
     const T: usize = 6;
+    const NUM_WITNESS: usize = 5;
+    /// When the number of fold rounds increases, `K` must be increased too
+    /// as the number of required rows in the table grows.
+    const NUM_OF_FOLD_ROUNDS: usize = 3;
+    /// 2 ^ K is count of table rows in [`TableData`]
+    const K: u32 = 19;
```

Comment:
Does it mean if we fold 3 times, we have to at least 2^19 rows here?

Reply:
This is folding of `W` only

Reply:
I wouldn't take these parameters very seriously yet, as there are a lot of test conventions there, but when we finish testing the whole folding it will be clear how much `K` we need

Reply:
I suggest to reduce the number of K until it just works. This will also save the unit test time.

Reply:
It won't work for 18 already, it's the minimum possible, I just said that you should take into account that not only fold_W itself, but also on-circuit decomposition in MAX_BITS for each iteration.

Reply:
yeah, that's my question in the beginning. "Do we need at least 2^19 rows here"